src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -151,11 +158,18 @@ private static void usage() {
         + "   to start a server, you can optionally password protect the game using triplea.server.password=foo");
   }
 
-  public static void main(final String[] args) {
+  public static void main(String[] args) {
+    if( args == null ) {
```

Comment:
I prefer using the nonnull road, with some static code analyzers (such as findbug), as it does not fills the code with redundant checks every step, still provide a warning for such cases and allows writing short and concise code